### ISSUE TITLE
Create the Temperature.js sub-component for DiagnosticContainer.js

### ISSUE THREAD

**Issue by tesla809** [COVID-19-electronic-health-system/Corona-tracker#22]
**Task**
Use the mockup in the design directory to create a component that houses the Average Temperature display, called Temperature.js

This will be a sub-component of DiagnosticContainer.js as per design/wireframes/CORONATRACKER UI - #17

**What done looks like**
Add comments to explain code.
We have a Temperature.js component in /components.
We have a Temperature.css file in /css to style the Temperature.js sub-component.

**Design considerations**
We have decided to use regular CSS files to keep things simple for now. This allows for more contributors and less time to upskill to contribute.

Once MVP is done the team can discuss if we want to transition it to styled-components or something like CSS modules.

**Folder paths**
CSS is located in /client/src/css
Components are located in /client/src/components

**Style guide for CSS files:
Naming convention:**
Component -> Temperature.js
CSS file -> Temperature.js

**To avoid naming collisions:**
prefix class names with the name of component and link with kebab case.
Class name -> .Temperature-header { ... }

**How to handle state:**
If you can use redux, use it.
Create action and reducer with an appropriate name.

**Testing:**
No need to worry about testing for now.
If you choose to add tests, please place in client/src/tests

**Comment by johnemurphy1** [comment]
Question on the average temperature. Is that taking the temperatures tracked on Chart.js, for example, and calculating a number or will it be pulling from somewhere else?

**Comment by SomeMoosery** [comment]
My guess is that we'll have a database entry `temperature: Number` that will store the temperature of a specific user for each "checkup" they complete (each time they fill out the form). 

I'll leave implementation details up to the those assigned to both `Chart.js` and this ticket, as there's many ways to go about this. I haven't dug too deeply into how you're all implementing the UI code, but I'd imagine when rendering the `DiagnosticComponent` component, you could do one API call and store an array of all `checkup->temperature`s for a given user. You could pass this array into `Chart.js` in order to plot each temperature, and then sum up the temperatures and pass the average into `Temperature.js`. 

This is just one idea though - you could just as easily do two API calls, one within 'Temperature.js` and another within `Chart.js` and manipulate the data within each component there.

**Comment by SomeMoosery** [comment]
Closing as #85 created the component with a default value. I'll open a new ticket for getting this data associated with each user from the database now that we have the scaffolding merged into master
